src/kiota/Rpc/Server.cs

```diff
@@ -80,7 +81,7 @@ private static async Task<bool> GenerateClientAsync(GenerationConfiguration conf
     {
         using var fileLogger = new FileLogLogger<KiotaBuilder>(config.OutputPath, LogLevel.Warning);
         var logger = new AggregateLogger<KiotaBuilder>(globalLogger, fileLogger);
-        return await new KiotaBuilder(logger, config, httpClient, true).GenerateClientAsync(cancellationToken);
+        return await new KiotaBuilder(logger, config, httpClient, IsConfigPreviewEnabled.Value).GenerateClientAsync(cancellationToken);
```

Comment:
QQ: Thinking about this, I'm sort of unsure if we really need to use a config here. Given that the extension is essentially moving to using workspace file. Would it be correct to assume that we can just say the RPC server only works with workspace files?

Reply:
This is a good question I don't have a definitive answer to. We have a couple of partners (Red Hat, other teams at Microsoft) who have taken a dependency on the server, or are thinking about doing so.
While we made it clear we don't guarantee the contract for the API itself, I want to maintain some level of compatibility with the lock mode for the time being so they are not backed into a corner.

Reply:
Thanks for the insight there.